server/src/main/java/org/opensearch/common/xcontent/JsonToStringXContentParser.java

```diff
@@ -0,0 +1,269 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
```

Comment:
The Elasticsearch related copyright notice is not needed (we didn't copy this code, right?)

Reply:
removed license head in the commit [Removed license head and add tests for wildcard query](https://github.com/opensearch-project/OpenSearch/pull/6507/commits/83a6b09af349fdddb48c206ecd049e1f72c92d10)